### PR TITLE
About section on the website + Styling improvements

### PR DESCRIPTION
**Pull request by filipemcarvalho**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
-A group of friends that decided to contribute to the community by starting to organize events where we could gather everyone interested in all the topics regarding testing. 
-Check our events <a href="https://portotestersmeetup.eventbrite.com" target="_blank">here</a>.
+
+# About
+
+<p>
+  A group of friends that decided to contribute to the community by starting to organize events where we could gather everyone interested in all the topics regarding testing.
+</p> 
+<p>
+  We start with one or two presentations and then discuss these and other themes afterwards along with a coffee break provided by the host company.
```
Comment (FabioCB) [inline index.md]: What do you think starting with "At each event".

File: README.md [hunk 2]
```diff
@@ -16,7 +20,7 @@ Note: **We will only accept PRs previously discussed in the issues section**
 1. Open the url of your fork of the repository - example: https://github.com/yourgithubusername/PTM_PAGE
 2. Go to the settings of the repository
 3. Open the options and scroll until Github Pages
-4. Select the theme (Cayman theme is the one used by the website)
+4. Select the theme Cayman
```
Comment (FabioCB) [inline README.md]: Also select master branch

**Review by FabioCB — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (filipemcarvalho) [reply README.md]: done

File: index.md (hunk 1 above)
Comment (filipemcarvalho) [reply index.md]: done

File: index.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
-A group of friends that decided to contribute to the community by starting to organize events where we could gather everyone interested in all the topics regarding testing. 
-Check our events <a href="https://portotestersmeetup.eventbrite.com" target="_blank">here</a>.
+
+# About
+
+<p>
+  A group of friends that decided to contribute to the community by starting to organize events where we could gather everyone interested in all the topics regarding testing.
+</p> 
+<p>
+  At each event we start with one or two presentations and then discuss these and other themes afterwards along with a coffee break provided by the host company.
```
Comment (FabioCB) [inline index.md]: Remove afterwards or replace "and then" with "and afterwards"

**Review by FabioCB — CHANGES_REQUESTED** [review]

**Review by FabioCB — APPROVED** [review]

File: index.md (hunk 3 above)
Comment (filipemcarvalho) [reply index.md]: done

**Pull request merged**